web/core/components/home/widgets/recents/page.tsx

```
@@ -0,0 +1,63 @@
+      itemLink=""
+      title={""}
+      prependTitleElement={
+        <div className="flex flex-shrink-0 items-center justify-center rounded-md gap-4 ">
+          <div className="flex flex-shrink-0 items-center justify-center rounded gap-4 bg-custom-background-80 w-8 h-8">
+            <>
+              {pageDetails?.logo_props?.in_use ? (
+                <Logo logo={pageDetails?.logo_props} size={16} type="lucide" />
+              ) : (
+                <FileText className="h-4 w-4 text-custom-text-300" />
+              )}
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)